src/components/Address/Address.js

```diff
@@ -250,14 +252,18 @@ export default function Address({
             setActiveSort={setActiveSort}
             url={url}
           />
-          <Sorting
-            activeSort={activeSort}
-            setActiveSort={setActiveSort}
-            activeFilter={domainType}
-          />
 
-          {domainType === 'registrant' && (
-            <>
+          <div
+            className={css`
```

Comment:
Let's change this to a styled component. I've used it once or twice in the project before, but generally we use styled-components to reduce the bloat in the render function

Reply:
Removed inline css + replaced w. new 'actions' grid item approach to resolve wrapping issue that @makoto discovered. See `Address.js` & `RenewAll.js` for details in:

* https://github.com/ensdomains/ens-app/pull/776/commits/f88810d398bfe4ad49d2a55897ba20fdab07d9c4
* https://github.com/ensdomains/ens-app/pull/776/commits/fec5670394badd68c8389674a4fa568873868cca